.travis.yml

```diff
@@ -18,6 +18,10 @@ jobs:
 
 install:
   - sudo apt-get install moreutils # make ts available
+  - sudo add-apt-repository -y ppa:wireguard/wireguard # add WireGuard support
```

Comment:
If you do these installs in the base container images it can be done once at image-building time, vs at each runtime.

https://github.com/submariner-io/submariner/tree/master/package
https://github.com/submariner-io/submariner/blob/master/package/Dockerfile.dapper-base

Reply:
It is a Linux kernel module, it is should be installed on the VM running the e2e tests.

Reply:
Oh that makes sense, okay thanks for clarifying.